daemon/algod/api/server/v2/generated/types.go

```diff
@@ -27,9 +27,7 @@ type Account struct {
 	// \[apar\] parameters of assets created by this account.
 	//
 	// Note: the raw account uses `map[int] -> Asset` for this type.
-	CreatedAssets *[]Asset `json:"created-assets,omitempty"`
-
-	// AccountParticipation describes the parameters used by this account in consensus protocol.
+	CreatedAssets *[]Asset              `json:"created-assets,omitempty"`
```

Comment:
why did you deleted the description for AccountParticipation ?

Reply:
I'm not sure why some comments were deleted. They're still in the spec. I'll need to look into this more.

Reply:
Found the issue, the codegen tool seems to need the file extension to be `yml` even if it's actually a `json` file. For now I've left the file extension as `yml` so that the comments get generated.